### PR TITLE
Add MODELTRANS_DEFAULT_LANGUAGE setting

### PR DESCRIPTION
**Pull request by jacobwegner**
See use case in #116.

I worked with the existing test suite using the following patch and verified that everything still passed as expected:

```diff
diff --git a/tests/app/settings.py b/tests/app/settings.py
index a0adb93..5229cd3 100644
--- a/tests/app/settings.py
+++ b/tests/app/settings.py
@@ -90,8 +90,8 @@ DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 # Internationalization
 # https://docs.djangoproject.com/en/1.11/topics/i18n/
 
-LANGUAGE_CODE = "en"
-
+LANGUAGE_CODE = "en-US"
+MODELTRANS_DEFAULT_LANGUAGE = "en"
 MODELTRANS_AVAILABLE_LANGUAGES = ("nl", "de", "fr")
 
 TIME_ZONE = "UTC"
```

I also added `DefaultLanguageConfTest` to verify that settings override got picked up.

Please let me know if there are any additional tests / documentation that would be required.

### CODE REVIEW

**Review by jieter — COMMENTED** [review]
The use case seems to be fair, and the implementation doesn't add much complexity 👍 

I think we should add this setting to the documentation. Can you extend the PR with documentation for this setting?

**Comment by jacobwegner** [comment]
@jieter Thanks for the feedback.  I did a pass to update the documentation and link to #116 for more context.

File: docs/pages/settings.rst [hunk 1]
```diff
@@ -4,13 +4,25 @@ Settings Reference
...
+
 ``MODELTRANS_AVAILABLE_LANGUAGES``
 ----------------------------------
 A list of language codes to allow model fields to be translated in. By default,
 the language codes extracted from django's `LANGUAGES setting <https://docs.djangoproject.com/en/stable/ref/settings/#languages>`_.
 
 Note that
- - the default language, defined in django's `LANGUAGE_CODE setting <https://docs.djangoproject.com/en/stable/ref/settings/#language-code>`_,
+ - the default language, defined in `MODELTRANS_DEFAULT_LANGUAGE`,
    should not be added to this list (will be ignored).
```
Comment (jieter) [inline docs/pages/settings.rst]:
```suggestion
 - the default language, defined in `MODELTRANS_DEFAULT_LANGUAGE` should not be added to this list (will be ignored).
```

**Review by jieter — COMMENTED** [review]
One last nitpick, and a test failure (slightly annoying false positive in the spell check, "middlewares" need to be added to https://github.com/zostera/django-modeltrans/blob/master/docs/spelling_wordlist.txt)  

I'll merge and create a new release after these are resolved...

**Review by jieter — APPROVED** [review]
Thanks!

**Pull request merged**